site/content/en/docs/Reference/API/components.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "components"
+linkTitle: "components"
+type: docs
+weight: 5
+description: >
+    Compose kustomizations.
+---
+
+Please check out the existing [components guide](/guides/config_management/components/) for explanation with examples. 
```

Comment:
Suggest updating this link path to `/docs/concepts/components/`.

Reply:
I added [this content](https://kubectl.docs.kubernetes.io/guides/config_management/components/) instead of referencing it because I don't think that the page will be added to our new website

https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/9d126f6dd8be627fc16fd19890e02632e9d528ce

I also added the reference to the `/docs/concepts/components/` page as you suggested.

https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/9d126f6dd8be627fc16fd19890e02632e9d528ce#diff-b6a9964b549b4e0db09ed4bebe9050677517c11321bf3a4dc754aec15e319e68R20

Reply:
As mentioned in this https://github.com/kubernetes-sigs/kustomize/pull/5313#pullrequestreview-1640692415, because this Components page is going to eventually live under Concepts, can we remove the extra "and the [components concept](/docs/concepts/components/) page"?